templates/single-tapestry.php

```diff
@@ -29,29 +29,39 @@ function addNonceToScript()
 
 get_header(); ?>
 
+<?php require "utility/utility.php";?>
```

Comment:
I'd prefer if you didn't put one file in a directory, so just put utility.php into the templates directory maybe.

Reply:
moved to a single utility.php folder